src/Trace/Integrations/PDO.php

```diff
@@ -71,7 +75,7 @@ public static function load()
     public static function handleQuery($pdo, $query)
     {
         return [
-            'attributes' => ['query' => $query],
+            'attributes' => ['db.statement' => $query],
```

Comment:
Need to understand these nomenclature changes.

Reply:
this is from opentracing conventions  - https://opentracing.io/specification/conventions/